### PR TITLE
Lotus v1.10.0

### PR DESCRIPTION
**Pull request by arajasek**
H Y P E R D R I V E

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
+
+Other improvements including better compressed snapshots, faster premigrations, and improved chain exports.
+
+## WindowPost base fee burn
+
+Included in the HyperDrive upgrade is [FIP-0015](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0015.md) which eliminates the special-case gas treatment of `SubmitWindowedPoSt` messages that was introduced in [FIP-0009](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0009.md). Although `SubmitWindowedPoSt` messages will be relatively cheap, thanks to the introduction of optimistic acceptance of these proofs in [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md), storage providers should pay attention to their `MaxWindowPoStGasFee` config option: too low and PoSts may not land on chain; too high and they may cost an exorbitant amount!
+
+## Changelog
+
+TODO
```
Comment (arajasek) [inline CHANGELOG.md:41]: @jennijuju pls provide

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,22 +1,44 @@
...
 Note that this release is built on top of Lotus v1.9.0. Enterprising users can use the `master` branch of Lotus to get the latest functionality, including all changes in this release candidate.
 
-A detailed changelog will be included in a later release candidate.
+## Proof batching and aggregation
+
+FIPs [0008](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0008.md) and [0013](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md) combine to allow for a significant increase in the rate of onboarding storage on the Filecoin network. It is hoped that this will lead to more useful data being stored on the network, reduced network congestion, and lower network base fee.
+
+### Projected state tree growth
+
+As a result of the accelerated onboarding of storage onto the network, it is possible
```
Comment (arajasek) [inline CHANGELOG.md:23]: @Stebalien pls provide

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,22 +1,44 @@
...
+
+### Future improvements
+
+Various Lotus improvements are planned moving forward to mitigate the effects of the growing state tree size. The primary improvement is the [Lotus splitstore](LINK TO DISCUSSION), which will soon be enabled by default. The feature allows for [online garbage collection](https://github.com/filecoin-project/lotus/issues/6577) for nodes that do not seek to maintain full chain and state history, thus eliminating the need for users to delete their datastores and sync from snapshots.
```
Comment (arajasek) [inline CHANGELOG.md:31]: @jennijuju Can you fill in the best link here?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,22 +1,138 @@
 # Lotus changelog
 
-# 1.10.0-rc6 / 2021-06-18
+# 1.10.0 / 2021-06-23
 
-> Note: If you are running a lotus miner, check out the doc [here](https://docs.filecoin.io/mine/lotus/miner-configuration/#precommitsectorsbatch) for new lotus miner configurations explanations of the new features!
-
-This is the 6th release candidate for Lotus v1.10.0, an upcoming mandatory release of Lotus that will introduce Filecoin network v13. Use this release for syncing with the [reset calibration net](https://github.com/filecoin-project/community/discussions/74#discussioncomment-885580). In addition, included in the new network version are the following FIPs:
+This is a mandatory release of Lotus that introduces Filecoin network v13, codenamed the HyperDrive upgrade. The Filecoin mainnet will upgrade, which is 2021-06-30T22:00:00Z. The network upgrade introduces the following FIPs:
```
Comment (momack2) [inline CHANGELOG.md:5]: you're missing an epoch

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,22 +1,138 @@
...
 - [FIP-0015](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0015.md): Revert FIP-0009(Exempt Window PoSts from BaseFee burn)
 
-This release candidate does not set the upgrade epochs for mainnet, but does set the upgrade epoch for the calibration network to 321519.
-
 Note that this release is built on top of Lotus v1.9.0. Enterprising users can use the `master` branch of Lotus to get the latest functionality, including all changes in this release candidate.
 
-A detailed changelog will be included in a later release candidate.
+## Proof batching and aggregation
+
+FIPs [0008](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0008.md) and [0013](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md) combine to allow for a significant increase in the rate of onboarding storage on the Filecoin network. It is hoped that this will lead to more useful data being stored on the network, reduced network congestion, and lower network base fee.
```
Comment (momack2) [inline CHANGELOG.md:17]:
```suggestion
FIPs [0008](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0008.md) and [0013](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md) combine to allow for a significant increase in the rate of onboarding storage on the Filecoin network. This aims to lead to more useful data being stored on the network, reduced network congestion, and lower network base fee.
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,22 +1,138 @@
...
+
+**Check out the documentation [here](https://docs.filecoin.io/mine/lotus/miner-configuration/#precommitsectorsbatch) for details of the new Lotus miner sealing config options, [here](https://docs.filecoin.io/mine/lotus/miner-configuration/#fees-section) for fee config options, and explanations of the new features.**
```
Comment (momack2) [inline CHANGELOG.md:19]:
```suggestion
**Check out the documentation [here](https://docs.filecoin.io/mine/lotus/miner-configuration/#precommitsectorsbatch) for details on the new Lotus miner sealing config options, [here](https://docs.filecoin.io/mine/lotus/miner-configuration/#fees-section) for fee config options, and explanations of the new features.**
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,22 +1,138 @@
...
+
+Note: 
+  - We recommend to keep `PreCommitSectorsBatch` as 1.
+  - We recommend miners to set `PreCommitBatchWait` lower than 30 hours.
+  - We recommend miners to set a longer `CommitBatchSlack` and  `PreCommitBatchSlac` to prevent message failures due to expirations.
```
Comment (momack2) [inline CHANGELOG.md:24]: why is slack spelled differently in these two different messages?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,22 +1,138 @@
...
+
+### Projected state tree growth
+
+In order to validate the Hyperdrive changes, we wrote a simulation to seal as many sectors as fast as possible assuming the same number and mix of 32GiB and 64GiB miners as the current network.
```
Comment (momack2) [inline CHANGELOG.md:28]:
```suggestion
In order to validate the Hyperdrive changes, we wrote a simulation to seal as many sectors as quickly as possible, assuming the same number and mix of 32GiB and 64GiB miners as the current network.
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,22 +1,138 @@
...
+
+Given these assumptions:
+
+- We'd expect a network storage growth rate of around 530PiB per day.
```
Comment (momack2) [inline CHANGELOG.md:32]:
```suggestion
- We'd expect a network storage growth rate of around 530PiB per day. 😳 🎉 🥳 😅
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,22 +1,138 @@
...
+- We'd expect network bandwidth dedicated to `SubmitWindowedPoSt` to grow by about 0.02% per day.
+- We'd expect the [state-tree](https://spec.filecoin.io/#section-systems.filecoin_vm.state_tree) to grow by 1.6GiB per day.
```
Comment (momack2) [inline CHANGELOG.md:34]:
```suggestion
- We'd expect the [state-tree](https://spec.filecoin.io/#section-systems.filecoin_vm.state_tree) to grow by 1.16GiB per day.
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,22 +1,138 @@
...
+   - Nearly all of the state-tree growth is expected to come from new sector metadata.
+- We'd expect the daily lotus datastore growth rate to increase by about 10-15%.
```
Comment (momack2) [inline CHANGELOG.md:36]:
```suggestion
- We'd expect the daily lotus datastore growth rate to increase by about 10-15% (from current ~21GiB/day).
```

File: CHANGELOG.md [hunk 12]
```diff
@@ -1,22 +1,138 @@
...
+
+### Hardware requirements and suggestions
+
+As the size of a single state tree grows, so will the size of the minimal datastore needed to sync the blockchain. The Filecoin protocol requires any node validating the chain to have the last 1800 state trees (2 * `ChainFinality`). Depending on how fast storage providers seal new sectors, this could get as large as AMOUNT in 6 months, and grow as fast as 20 GB per day. This increases the necessary hardware requirements to validate the Filecoin blockchain, but can be supported by images such as [Amazon EC2](https://aws.amazon.com/ec2/instance-explorer/?ec2-instances-cards.sort-by=item.additionalFields.category-order&ec2-instances-cards.sort-order=asc&awsf.ec2-instances-filter-category=*all&awsf.ec2-instances-filter-processors=*all&awsf.ec2-instances-filter-accelerators=*all&awsf.ec2-instances-filter-capabilities=additional-capabilities%23instance-storage&ec2-instances-cards.q=ssd&ec2-instances-cards.q_operator=AND&awsm.page-ec2-instances-cards=1) and [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-types).
```
Comment (momack2) [inline CHANGELOG.md:41]:
```suggestion
As the size of a single state tree grows, so will the size of the minimal datastore needed to sync the blockchain. The Filecoin protocol requires any node validating the chain to have the last 1800 state trees (2 * `ChainFinality`). Depending on how fast storage providers seal new sectors, the state-tree could get as large as 224GiB in 6 months, and grow as fast as 20 GB per day.

This increases the necessary hardware requirements to validate the Filecoin blockchain, but can be supported by images such as any of these [Amazon EC2 instances](https://aws.amazon.com/ec2/instance-explorer/?ec2-instances-cards.sort-by=item.additionalFields.category-order&ec2-instances-cards.sort-order=asc&awsf.ec2-instances-filter-category=*all&awsf.ec2-instances-filter-processors=*all&awsf.ec2-instances-filter-accelerators=*all&awsf.ec2-instances-filter-capabilities=additional-capabilities%23instance-storage&ec2-instances-cards.q=ssd&ec2-instances-cards.q_operator=AND&awsm.page-ec2-instances-cards=1) or [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-types).
```


Make this better please

**Review by Stebalien — APPROVED** [review]

**Review by jennijuju — APPROVED** [review]

**Comment by momack2** [comment]
LG

**Pull request merged**